.gitignore

```diff
@@ -1,32 +1,25 @@
-.env
+._*
```

Comment:
# Reviewer Note

Had to tweak the gitignore to change for the build output, sorted the list and cleaned it up while here.